From e4eb10ca24d263620cbae7bb6671e86ae0f8172b Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@vates.fr>
Date: Mon, 21 Oct 2024 17:49:20 +0200
Subject: [PATCH] Use `for session` instead of `for e`

Avoid mypy error:
```
error: Assignment to variable "e" outside except: block  [misc]
```

Signed-off-by: Ronan Abhamon <ronan.abhamon@vates.fr>
---
 drivers/iscsilib.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/iscsilib.py b/drivers/iscsilib.py
index e77d17d2..e54de1bc 100644
--- a/drivers/iscsilib.py
+++ b/drivers/iscsilib.py
@@ -535,8 +535,8 @@ def _checkAnyTGT():
     except Exception as e:
         util.SMlog("%s failed with %s" % (cmd, e.args))
         stdout = ""
-    for e in filter(match_session, stdout.split('\n')):
-        iqn = e.split()[-1]
+    for session in filter(match_session, stdout.split('\n')):
+        iqn = session.split()[-1]
         if not iqn in rootIQNs:
             return True
     return False
